docs/STM32_Ports.md

```diff
@@ -1,9 +1,29 @@
-# List of STM32 Port Names (from *_FLASH.ld files)
+# List of STM32 Port Names
+
+## L4
```

Comment:
idt you need to be listing out PROJECT_TARGET each time just the corresponding target argument for each and just have a blurb at the top confirming how to set the argument in a command

Reply:
Also maybe saying you can view target specific configs in `stm/` would be nice. Not sure if its on here somewhere else.